CommonTools/scripts

```diff
@@ -1 +1 @@
-Subproject commit 73e6dc7965916e2be119e942e29716cf1fb9098c
+Subproject commit 60e5a4c131a28a3b03c63f596f8f01f0fd8f4edb
```

Comment:
I followed your instruction, but still not go away... 
    $ git branch
    * (detached from 60e5a4c)
    master
Is it alright?